Doc/library/pprint.rst

```diff
@@ -81,25 +87,43 @@ The :mod:`pprint` module defines one class:
 
 The :mod:`pprint` module also provides several shortcut functions:
 
-.. function:: pformat(object, indent=1, width=80, depth=None, *, compact=False)
+.. function:: pformat(object, indent=1, width=80, depth=None, *, \
+                      compact=False, sort_dicts=True)
 
    Return the formatted representation of *object* as a string.  *indent*,
-   *width*, *depth* and *compact* will be passed to the :class:`PrettyPrinter`
-   constructor as formatting parameters.
+   *width*, *depth*, *compact* and *sort_dicts* will be passed to the
+   :class:`PrettyPrinter` constructor as formatting parameters.
+
+   .. versionchanged:: 3.8
+      Added the *sort_dicts* parameter.
 
    .. versionchanged:: 3.4
       Added the *compact* parameter.
 
 
+.. function:: pp(object, *args, sort_dicts=False, **kwargs)
+
+   Prints the formatted representation of *object* followed by a newline.
+   If *sort_dicts* is false (the default), dictionaries will be displayed with
+   their keys sorted in insertion order, otherwise they will be sorted
+   alphabetically. *args* an *kwargs* will be passed to :function:`pprint` as
```

Comment:
Remove the `:function:`

Reply:
Do you think I should change it to `:func:`?

Reply:
Yeah

Reply:
For some reason Travis is angry with :function: